databricks/koalas/frame.py

```diff
@@ -2146,8 +2146,9 @@ def apply_batch(self, func, args=(), **kwds):
             # If schema is inferred, we can restore indexes too.
             internal = kdf._internal.with_new_sdf(sdf)
         else:
-            return_schema = infer_return_type(original_func).tpe
-            is_return_dataframe = getattr(return_sig, "__origin__", None) == ks.DataFrame
+            return_tpe = infer_return_type(original_func)
```

Comment:
How about `return_type` rather `return_tpe` since the method name that return this variable is `infer_return_type`?